### PR TITLE
DOC-4888: Beta2 merge 2023 03 28

### PR DESCRIPTION
**Pull request by nadinequinr3**
Forward merge for DOC-4888 from Beta 2 branch on into release/os/5.0

### CODE REVIEW

File: data/config-schema/bin/main/net/corda/schema/configuration/flow/1.0/corda.flow.json [hunk 1]
```diff
@@ -0,0 +1,112 @@
+{
```
Comment (vkolomeyko) [inline data/config-schema/bin/main/net/corda/schema/configuration/flow/1.0/corda.flow.json]: Is this new file expected?

File: data/config-schema/bin/main/net/corda/schema/configuration/sandbox/1.0/corda.sandbox.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
+{
```
Comment (vkolomeyko) [inline data/config-schema/bin/main/net/corda/schema/configuration/sandbox/1.0/corda.sandbox.json]: Is this new file expected?

**Review by vkolomeyko — COMMENTED** [review]
Odd files found.

**Comment by vkolomeyko** [comment]
Gentle reminder not to squash commits when merging this PR.

File: data/config-schema/bin/main/net/corda/schema/configuration/flow/1.0/corda.flow.json (hunk 1 above)
Comment (nadinequinr3) [reply data/config-schema/bin/main/net/corda/schema/configuration/flow/1.0/corda.flow.json]: Removed. Thanks.

File: data/config-schema/bin/main/net/corda/schema/configuration/sandbox/1.0/corda.sandbox.json (hunk 2 above)
Comment (nadinequinr3) [reply data/config-schema/bin/main/net/corda/schema/configuration/sandbox/1.0/corda.sandbox.json]: Removed. Thanks.

**Review by vkolomeyko — APPROVED** [review]
LGTM

**Pull request merged**